### DISCUSSION TITLE
Convert imagemagick command to PHP code

### DISCUSSION THREAD

**Discussion by keyurshah070** [ImageMagick/ImageMagick#2243] [Help]
Hello I have below command 

magick image.png -channel RGB -colorspace gray -auto-level +level-colors '#ff00ff',  result.png

and I want to convert into PHP code so how can I do this?

**Comment by urban-warrior** [comment]
Do you mean you want to use IMagick?  Otherwise, use the PHP exec() call which can run the ImageMagick command-line within PHP.
